tgui/packages/tgui-dev-server/webpack.js

```diff
@@ -32,14 +32,21 @@ class RspackCompiler {
     const requireFromRoot = createRequire(import.meta.dirname + '/../../..');
     /** @type {typeof import('@rspack/core')} */
     const rspack = await requireFromRoot('@rspack/core');
+
     const createConfig = await requireFromRoot('./rspack.config.cjs');
+    const createDevConfig = await requireFromRoot('./rspack.config-dev.cjs');
+
     const config = createConfig({}, options);
+    const devConfig = createDevConfig({}, options);
+
+    const mergedConfig = { ...config, ...devConfig };
```

Comment:
see here where i just overwrite the regular config with the dev server config, which uses https://rspack.dev/plugins/rspack/css-extract-rspack-plugin